pkg/e2e/openshift/osdclusterready.go

```diff
@@ -0,0 +1,57 @@
+// Package openshift contains the OpenShift extended test suite.
+package openshift
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/openshift/osde2e-common/pkg/clients/openshift"
+	"github.com/openshift/osde2e/pkg/common/label"
+	batchv1 "k8s.io/api/batch/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/watch"
+	"k8s.io/client-go/kubernetes"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+var suiteName = "OSD Cluster Ready"
+var timeout = "30m"
+var namespace = "openshift-monitoring"
+var jobname = "osd-cluster-ready"
+
+// Assert osd-cluster-ready status as a e2e spec
+var _ = ginkgo.Describe(suiteName, ginkgo.Ordered, label.OCPNightlyBlocking, func() {
+	var (
+		duration time.Duration
+		k8s      *openshift.Client
+	)
+
+	ginkgo.BeforeAll(func(ctx context.Context) {
+		log.SetLogger(ginkgo.GinkgoLogr)
+		var err error
+		k8s, err = openshift.New(ginkgo.GinkgoLogr)
+		Expect(err).ShouldNot(HaveOccurred(), "Unable to setup k8s client")
+	})
+
+	ginkgo.It("should verify cluster is ready", func(ctx context.Context) {
+		var logs string
+		duration, err := time.ParseDuration(timeout)
+		Expect(err).ShouldNot(HaveOccurred(), "failed to parse duration")
```

Comment:
Is there a reason we don't just make a `var timeout = 30 * time.Minute` object? It is already a constant at the top of the file